### ISSUE TITLE
System navbar on Samsung doesn't respect website color

### ISSUE THREAD

**Issue by emvaized** [bromite/bromite#2416]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.
- [X] This is a bug report about the Bromite browser; not the website nor F-Droid nor anything else.

### Can the bug be reproduced with corresponding Chromium version?

No

### Bromite version

106.0.5249.72

### Device architecture

arm64

### Android version

12.0

### Device model

Samsung Galaxy Note 10 (SM-N9700)

### Changed flags

no flags changed

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

Yes

### Is this bug caused by the adblocker?

No

### Is this bug a crash?

No

### Describe the bug

Bromite provides useful feature to move the toolbar to the bottom. However, while toolbar gets colored automatically depending on website theme (as well as statusbar on top), system navbar does not — resulting in a not very pleasant visual experience: colored toolbar feels unanchored an out-of-place when navbar always remains white (please see the screenshot). 

Same behavior can be observed in Chromium — but I believe it requires a special patch for Bromite, as Chromium does not have an option for bottom toolbar, and thus it doesn't need navbar coloring that much to visually match the toolbar.

### Steps to reproduce the bug

1. Set the option to move toolbar to bottom
2. Visit some page with dynamic color theme, such as [this page](https://github.com/bromite/bromite) on GitHub

### Expected behavior

Bottom toolbar and navbar should always be colored the same for better visual consistency

### Screenshots

![Screenshot_20221029-182143_Bromite](https://user-images.githubusercontent.com/37851576/198841881-ccf2b309-f17a-48aa-9769-03fb8dcbe459.jpg)

**Comment by uazo** [comment]
>> Can the bug be reproduced with corresponding Chromium version?
>
> No

actually I don't think so, in the sense that the colours of the bar are the same with the bar at the top, can you check?

in that case it would possibly be a matter of disabling the bar background change, i.e. adjusting it to the theme flag in the settings.

**Comment by emvaized** [comment]
> colours of the bar are the same with the bar at the top

You mean status bar? Yes, it's colored the same as action bar (the one with url and menu button). But the navigation bar on bottom always remains black/white, depending on system night mode. 
As an example of correct navbar coloring, please check [Infinity for Reddit](https://play.google.com/store/apps/details?id=ml.docilealligator.infinityforreddit)  – navigation bar gets colored correctly, the same as status bar and other panels:
<details>
<summary>Screenshot</summary>

![Screenshot_20221102-134004_Infinity](https://user-images.githubusercontent.com/37851576/199481174-032e43af-0e88-445a-8f9d-9ad8f19f1594.jpg)
</details>

**Comment by uazo** [comment]
no, what I am saying is if you detect this behavior, even with chromium:

![image](https://user-images.githubusercontent.com/29201891/199488004-70e94d30-1db4-405d-8f18-6b1017e27d44.png)

if you detect the bar below white it is an upstream bug

**Comment by emvaized** [comment]
Ah, I see what you mean! I addressed it in the main text by this statement: 

> Same behavior can be observed in Chromium — but I believe it requires a special patch for Bromite, as Chromium does not have an option for bottom toolbar, and thus it doesn't need navbar coloring that much to visually match the toolbar

**Comment by csagan5** [comment]
Fixed in `107.0.5304.114`.

**Comment by emvaized** [comment]
Great, thanks!

**Comment by emvaized** [comment]
There's also a bit of white line remaining between nav bar and toolbar in `107.0.5304.114` – but I'm not sure if it's worth any further attention or a new ticket... Maybe it's intentional?
![Screenshot_20221119-224822_Bromite](https://user-images.githubusercontent.com/37851576/202871159-614fd252-bea9-4d8a-8897-f908b9f0b798.jpg)

**Comment by uazo** [comment]
>  Maybe it's intentional?

it should be above, not below. i thought i moved it. I will check

**Comment by emvaized** [comment]
Great, thanks!

**Comment by emvaized** [comment]
@uazo 
While v109 is being in review, could you please check the line problem in your spare time? Not like it's something crucial, just a little bit annoying in everyday usage... Thanks
